csharp/src/Ice/TcpEndpoint.cs

```diff
@@ -171,7 +172,7 @@ internal TcpEndpoint(InputStream istr, Communicator communicator, Transport tran
             }
             else
             {
-                Timeout = Communicator.DefaultTimeout;
+                Timeout = 60000; // 60s timeout by default
```

Comment:
I think it would be cleaner to convert this Timeout into a Timespan like every other timeouts.